source/includes/_create_jackal_pcb.md

```diff
@@ -0,0 +1,39 @@
+## `/parts/create_jackal_pcb`
+
+> A POST request to this endpoint will create a new jackal pcb part on the system
+
+```python
+    url = "https://smartapi.bboxx.co.uk/v1/parts/create_jackal_pcb"
+    data = json.dumps({"imsi": "213514135171353",
+                       "imei": "010101234560005",
+                       "device_key": "4ab1ab3ab1ab7ab9ab1ab4ab",
+                       "manufacturer": "A Factory"})
+    headers = {"Content-Type": "application/json", "Authorization": "Token token=" + A_VALID_TOKEN}
+
+    r = requests.post(url=url, data=data, headers=headers)
+
+    print r.json()
+    >>> {"status": "success"
+         "message": "new pcb part and sim successfully created",
+         "data": {"pcb": {"part_type_id": 44,
+                          "properties": {"device_key": "4ab1ab3ab1ab7ab9ab1ab4ab",
+                                         "imei": "010101234560005",
+                                         "imsi": "213514135171353",
+                                         "serial_number": "HB000501-084032-G"},
+                          "serial_number": "010101234560005",
+                          "manufacturer": "A Factory"},
+                  "sim": {"part_type_id": 43,
+                          "serial_number": "213514135171353"}},
+     }
```

Comment:
The actual response is different. The documentation is meant to reflect the actual request/response, please update the documentation accordingly.